README.md

```diff
@@ -144,6 +146,51 @@ A `RequirementType` of `All` changes the traversal. First, if there are no filte
 
 In the above example, `FeatureW` specifies a `RequirementType` of `All`, meaning all of its filters must evaluate to true for the feature to be enabled. In this case, the feature will be enabled for 50% of users during the specified time window.
 
+#### Microsoft Feature Flag Schema
+
+The feature management library also supports the usage of [`Microsoft Feature Flag schema`](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureFlag.v1.1.0.schema.json) to declare feature flags.
+
+If you want to use the `Microsoft Feature Flag schema`, you will need to follow the Azure App Configuration Feature Management Configuration schema to set up feature flags. The schema can be found [here](https://github.com/Azure/AppConfiguration/tree/main/docs/FeatureManagement). The feature flag declarations should be listed in the `FeatureFlags` array under the `FeatureManagement` section, as shown in the example below:
+
+```json
+{
+    "FeatureManagement": {

```

Comment:
Looking at this I'm thinking, it's kind of odd that `FeatureManagement` and `FeatureFlags` have a separate naming convention than the property names of individual flags. Especially given that this is the `Microsoft Feature Management Schema` & the `Microsoft Feature Flag Schema`.

Seems it should be `feature_management` and `feature_flags`.

Reply:
@rossgrambo

Reply:
@jimmyca15 @rossgrambo I will take this. The json schema and the code base will both need to change.

Reply:
@jimmyca15 PR to let our code base to recognize snake_casing: #370